### ISSUE TITLE
Allow favorite/follow while in stream

### ISSUE THREAD

**Issue by martinoj2009** [Ippytraxx/gnome-twitch#109]
Don't mean to post a lot, but I really want to give my feedback as much as possible. I think there should be an option to follow the stream I'm watching while watching the stream. 

**Version:**
0.2.0

**Screenshot:**
![twitch](https://cloud.githubusercontent.com/assets/6756628/14395915/9e78a0f2-fda2-11e5-95ec-38615c63891a.png)

**Comment by Ippytraxx** [comment]
Yes absolutely, I'm currently working on a popup that will act as a "channel page". It will show things like viewers, followers, info page and allow you to do things like favourite/follow etc.

**Comment by martinoj2009** [comment]
That's awesome! I can't wait to see how this thing develops over time :-D
